models/Include/cps/Base/Base_SimpSynchronousGenerator.h

```diff
@@ -0,0 +1,209 @@
+/* Copyright 2017-2021 Institute for Automation of Complex Power Systems,
```

Comment:
This is the class that you name as `ReducedOrderSynchronousGenerator` in your thesis (Fig 3.1), right? I find this term clearer, so I would also use it here in the code